src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -66,16 +74,150 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = props => {
     setVerifyAddressNow(false)
   }
 
+  // Once the user sees the address form, they should always see it.
+  const [forceNewAddressFormMode, setForceNewAddressFormMode] = useState(
```

Comment:
How about `isNewAddressFormMode, `setNewAddressFormMode`

Reply:
Not a hill I'm going to die on, but I prefer making explicit that we are **forcing** address form mode [here](https://github.com/artsy/force/pull/13246#discussion_r1421146403), since the default behavior would be to just show the saved addresses version once an address is saved. A more generic `is_` naming seems more confusing to me - why is it being set in a different file.

Actually I think this should be restructured to move the whole linked area's logic up into this component. The reason for putting it here was [this line](https://github.com/artsy/force/pull/13246/files#diff-71b8774db34fee1f35b7c4594e7226120f02c41e93065e5d7fb9551fd7543991R210) - new address form mode performs user address updates after saving details, while saved address modals do the updates first and update fulfillment details after. This inconsistency could obviously be improved too. But for now, I'm going to move this logic next to each other and it should read better.

Reply:
makes sense